packages/roosterjs-content-model-plugins/lib/tableEdit/editors/features/TableMover.ts

```diff
@@ -129,3 +157,212 @@ function isTableTopVisible(editor: IEditor, rect: Rect | null, contentDiv?: Node
 
     return true;
 }
+
+function setTableMoverCursor(editor: IEditor, state: boolean, type?: 'move' | 'copy') {
+    editor?.setEditorStyle(TABLE_MOVER_STYLE_KEY, state ? 'cursor: ' + type ?? 'move' : null);
+}
+
+function onDragStart(context: TableMoverContext, event: MouseEvent) {
+    context.onStart();
+
+    const { editor, table, div } = context;
+
+    setTableMoverCursor(editor, true, 'move');
+
+    // Create table outline rectangle
+    const trect = table.getBoundingClientRect();
+    const createElementData = {
+        tag: 'div',
+        style: 'position: fixed; user-select: none; border: 1px solid #808080',
+    };
+    const tableRect = createElement(createElementData, document) as HTMLDivElement;
+    tableRect.style.width = `${trect.width}px`;
+    tableRect.style.height = `${trect.height}px`;
+    tableRect.style.top = `${trect.top}px`;
+    tableRect.style.left = `${trect.left}px`;
+    div.parentNode?.appendChild(tableRect);
+
+    // Get current selection
+    const initialSelection = editor.getDOMSelection();
+
+    // Select first cell of the table
+    editor.setDOMSelection({
```

Comment:
Curious, instead of just select the first cell, how about we select the whole table?

Reply:
Oh I see you just want to get the selected table.